app/src/main/java/com/mho/training/features/splash/SplashActivity.kt

```diff
@@ -0,0 +1,21 @@
+package com.mho.training.features.splash
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import com.mho.training.features.main.MainActivity
+import com.mho.training.utils.startActivity
+
+class SplashActivity : AppCompatActivity() {
+
+    //region Override Methods & Callbacks
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        startActivity<MainActivity> {}
```

Comment:
Does startActivity wait some seconds to start MainActivity?

Reply:
The splash lasts until the mainly activity is ready.

https://medium.com/@sneyderangulo/como-implementar-splash-screen-correctamente-en-android-f6abcc592b4c